CHANGELOG.md

```diff
@@ -16,21 +18,21 @@ The command used was
 ./statick . --output-directory /tmp/x --level self_check --log info --timings
 ```
 
-package | name         | plugin_type | duration (v0.9.2) | duration (Unreleased)
-------- | ------------ | ----------- | --------------- | --------------
-statick | find files   | Discovery   |  9.2810         | 9.3033
-statick | black        | Tool        |  4.5896         | 0.1365
-statick | docformatter | Tool        |  3.7007         | 0.8105
-statick | isort        | Tool        |  4.2516         | 0.1088
-statick | mypy         | Tool        |  6.7002         | 1.8145
-statick | pydocstyle   | Tool        |  5.2146         | 0.8606
-statick | pyflakes     | Tool        |  4.2164         | 0.1893
-statick | pylint       | Tool        |  2.2831         | 2.2545
-statick | shellcheck   | Tool        |  0.0762         | 0.0352
-statick | uncrustify   | Tool        |  0.0001         | 0.0001
-statick | xmllint      | Tool        |  0.0039         | 0.0037
-statick | yamllint     | Tool        |  1.0534         | 0.1654
-Overall |              |             | 41.4780         | 15.8050
+package | name         | plugin_type | duration (v0.9.2) | duration (unreleased)
```

Comment:
should this `unreleased` be changed to reflect the release version in this PR?
```suggestion
package | name         | plugin_type | duration (v0.9.2) | duration (v0.9.3)
```

Reply:
I was planning to update the CHANGELOG after the release. I got the other numbers from the jobs related to the actual release, and we haven't had that job run yet. I would prefer to wait and be precise on which exact job we get the numbers from.